snuba/environment.py

```diff
@@ -75,15 +76,16 @@ def setup_logging(level: Optional[str] = None) -> None:
 
 def setup_sentry() -> None:
     sentry_sdk.init(
-        dsn=settings.SENTRY_DSN,
+        dsn="https://4d597b532d8a4e8facc7c369de789e55@o1301100.ingest.sentry.io/4505512925724672",
```

Comment:
Please do not hardcode the DSN. It should come from settings.

Reply:
Fat-fingered `commit -a`, sorry!